jkube-kit/resource/service/src/test/java/org/eclipse/jkube/kit/resource/service/DefaultResourceServiceTest.java

```diff
@@ -60,22 +55,22 @@ public class DefaultResourceServiceTest {
   private ResourceServiceConfig resourceServiceConfig;
   private DefaultResourceService defaultResourceService;
 
-  @Before
-  public void init() throws IOException {
-    targetDir = temporaryFolder.newFolder("target");
+  @BeforeEach
+  void init(@TempDir File temporaryFolder) {
+    targetDir = new File(temporaryFolder, "target");
```

Comment:
We should create/link an actual directory

Reply:
Got it